doc/files/package.json.md

```diff
@@ -23,7 +23,8 @@ Some rules:
 
 * The name must be less than or equal to 214 characters. This includes the scope for
   scoped packages.
-* The name can't start with a dot or an underscore.
+* The name can't start with a dot or an underscore if the package name does not
```

Comment:
I think this might be clearer as "The names of scoped packages can begin with a dot or an underscore. This is not permitted without a scope."